docs/checks.md

```diff
@@ -274,3 +274,11 @@ arg1='SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\DisableCAD'
 > **Note!**: Notice the single quotes `'` on the above argument! This means it's a _string literal_ in TOML. If you don't do this, you have to make sure to escape your slashes (`\` --> `\\`)
 
 > Note: You can use `SOFTWARE` as a shortcut to mean `HKEY_LOCAL_MACHINE\SOFTWARE`.
+
+**FileOwner**: pass if specified owner is the owner of the specified file
+
+```
+type='FileOwner'
+arg1='C:\test.txt'
+arg2='BUILTIN\Administrators'
```

Comment:
It would be better if they only put username, because command would still work

Reply:
What do you mean? Like no BUILTIN\ or computer name?

Reply:
Yeah

Reply:
But the output of `(Get-Acl FILENAME).Owner` is in that format

Reply:
![image](https://user-images.githubusercontent.com/52616034/88962585-7ed58200-d25b-11ea-9fb6-6dd11401cc99.png)

Reply:
But it is unnecessary as the computer cant have usernames with the same name, so no need of it

Reply:
Doesn't command output check for the exact string returned?

Reply:
I am pretty sure if you just do the name, it would still show it.

Reply:
I don't think so, should we just ask Shane?

Reply:
Lmao, @sourque plz confirm

Reply:
plz sir

Reply:
Yeah `commandOutput` should only pass if the output _exactly_ matches. I think this helps reduce mistakes. If we make them specify `BUITLIN`, it would allow some compatibility with a domain environment (I assume), but also make the check a bit more clumsy from the UX perspective. We also don't handle Administrators/Guest accounts very well (what if someone renames it)? But yeah I think it's good to have the full name.

Reply:
aight can someone approve then